index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <p id="para-1">Hello World we are software developers!</p>
```

Comment:
Semantically speaking this would fit better inside an h1-h6 tag instead of a <p> tag.

Reply:
- [Optional] Please try to always use a friendly phrase while giving a review, so as to motivate the student.

##### Good
- This is a polite suggestion: *"I think it would be better to use semantic tags"*
##### Bad 
- This reads as an order: *"Use semantic tag*